coding/cpp_style_guide.rst

```diff
@@ -788,25 +788,25 @@ Keeps inputs together.
 
 .. _style-guide-cpp-3-34:
 
-3-34. Uncertainty values associated with a variable SHOULD be suffixed by one of ``Var``, ``Cov``, ``Sigma``.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-There is no universal suffix for uncertainties; i.e. no ``Err`` suffix will be used.
-The cases that we have identified, and their appropriate suffixes, are:
+3-34. Uncertainty values associated with a variable SHOULD be suffixed by one of ``Err`` or``Cov``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Standard deviation: ``Sigma`` (not ``Rms``, as ``rms`` doesn't imply that the mean's subtracted)
-- Covariance: ``Cov``
-- Variance: ``Var``
+Following the DPDD, we use ``Err`` to specify error quantities (as standard errors), and ``Sigma`` to specify values inherent width of a distribution or function.
```

Comment:
Something is wrong here; perhaps replace "values" with "the"?  Furthermore, the mention of `Sigma` seems confusing when combined with the title, which fails to mention that term.  More explanation might help.

Reply:
I've reworded it slightly. Is that better? I discuss `Sigma` here because this section previously discussed it and it seems appropriate to have both of these in the same section.